### PR TITLE
[W33] evanxd: 8/15 - 8/19

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

File: evanxd.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+      - landed
     - Bug 1282465 - Reps: fix or remove recursive handling in ArrayRep and Obj rep
-      - review+
-    - Bug 1289062 - Order the Event's properties
-      - feedback+
+      - landed
   - Location Bar
     - Bug 1256074 - Always present a 'search' option, even location bar's contents are detected as URLs
-      - Looking for the code determine location bar treat the string as an URL or a keyword.
+      - After investigated, we could add two `richlistitem` elements(moz-action:visiturl and moz-action:searchengine ones) on the `#PopupAutoCompleteRichResult` panel when the `moz-action` is `visiturl`.
```
Comment (timdream) [inline evanxd.md]: 👍 

Instead of waiting for a needinfo, how about let's just submit a patch and set ui-review?

File: evanxd.md [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+      - The intermittent failures might be related with [`_searchLogins`][storage-json.js] method in `storage-json.js` file.
   - Form Autofill
-    - Survey works
-      - Read UX spec to understand what we want to do in the feature.
-      - Can learn related tech knowledge from [Autofill Forms][autofill-forms].
+    - Implemented a [workable prototype add-on][form-autofill] to autofill forms.
 
-[autofill-forms]: https://github.com/sarahavilov/Autofill-Forms
+[form-autofill]: https://github.com/evanxd/form-autofill
+[storage-json.js]: http://searchfox.org/mozilla-central/source/toolkit/components/passwordmgr/storage-json.js#282
```
Comment (timdream) [inline evanxd.md]: What's this?

File: evanxd.md [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
-      - Re-triggered the browser_capture_doorhanger.js test for 10 times. If they are all good, let's close the bug.
+      - The intermittent failures might be related with [`_searchLogins`][storage-json.js] method in `storage-json.js` file.
   - Form Autofill
-    - Survey works
-      - Read UX spec to understand what we want to do in the feature.
-      - Can learn related tech knowledge from [Autofill Forms][autofill-forms].
+    - Implemented a [workable prototype add-on][form-autofill] to autofill forms.
 
-[autofill-forms]: https://github.com/sarahavilov/Autofill-Forms
+[form-autofill]: https://github.com/evanxd/form-autofill
```
Comment (timdream) [inline evanxd.md]: You should no longer working on this as your official duty unless communicated otherwise.

Comment (evanxd) [inline evanxd.md]: Sure.

File: evanxd.md (hunk 1 above)
Comment (evanxd) [inline evanxd.md]: Sure, let's do it!

**Pull request merged**